src/runtime_src/tools/xclbinutil/Section.h

```diff
@@ -80,12 +81,12 @@ class Section {
   static enum FormatType getFormatType(const std::string & sFormatType);
   static bool supportsSubSections(enum axlf_section_kind &_eKind);
   static bool supportsSectionIndex(enum axlf_section_kind &_eKind);
+  static bool doesSupportAddFormatType(enum axlf_section_kind eKind, FormatType eFormatType);
```

Comment:
I guess you do not need to use `enum` everywhere like in C?

Reply:
Updated the entire xclbinutil codebase to no longer use the enum 'C Style' coding practice.